src/integrationTest/java/uk/gov/hmcts/darts/workflow/command/DeployRedisCommand.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.darts.workflow.command;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+
+import java.io.IOException;
+
+public class DeployRedisCommand implements Command {
+
+    private static GenericContainer<?> container;
+
+    static {
+        ImageFromDockerfile redisImage = new ImageFromDockerfile()
+            .withDockerfileFromBuilder(builder ->
+                                           builder
+                                               .from("redis:7.2.4-alpine")
```

Comment:
what happens if image not found? e.g. if version changes?

Reply:
we use public repo so it shouldn't not be there. It will always use the latest version.